.circleci/config.yml

```diff
@@ -19,8 +19,9 @@ jobs:
       - run:
           name: style-check
           command: |
-            pip install --progress-bar off --user -U flake8==3.7.1
+            pip install --progress-bar off --user -U flake8==3.7.1 pydocstyle==5.0.2
```

Comment:
Let's separate these into multiple `- run:` blocks, so that the dependency installation logs aren't mixed up with the output of `flake8` and `pydocstyle`. I would put the `pip` command into one block named "install dependencies" (to match the test template), then a second block named "flake8", then a third block named "pydocstyle". This will make it easier to tell what failures are occurring when reading the CI output.

Reply:
Sounds good!